environments/staging/terraform.yml

```diff
@@ -91,6 +91,13 @@ servers:
     az: "a"
     volume_size: 80
 
+  - server_name: "kafka0-staging"
+    server_instance_type: t3.medium
+    network_tier: "db-private"
```

Comment:
Should this be app-private?

Reply:
Nope, app-private is for the app layer, things that run the code that we write for commcarehq; kafka is in the db layer, it's a third-party service for general use that we rely on, analogous to the way we use redis, postgres, elasticsearch, couchdb, for different purposes.

Reply:
Is there a documentation to understand what these fields and their values mean?

Reply:
I have an explanation on this page: https://confluence.dimagi.com/display/internal/How+to+migrate+a+rackspace+machine+to+AWS
Explained in step 1 of 'Create and Provision the New AWS Machine'

Reply:
Thank you @pr33thi